app/components/message_threads_table_component.html.erb

```diff
@@ -1,13 +1,14 @@
 <div class="flex flex-col justify-stretch items-stretch gap-4 sm:p-4">
   <div class="flex flex-col justify-stretch items-stretch sm:rounded-md bg-white sm:border sm:border-gray-200" data-controller="form all-checkboxes">
     <%= render MessageThreadsBulkActionsComponent.new(ids: [], filter: @filter, filter_subscription: @filter_subscription, signable: Current.user.signer?) %>
-    <%= form_with url: bulk_actions_message_threads_path, data: { "form-target": "form", "all-checkboxes-target": "form" } do %>
+    <%= form_with url: bulk_actions_message_threads_path, data: { 'form-target': "form", 'all-checkboxes-target': "form" } do %>
```

Comment:
tie dvojite " nedavame vsade?

Reply:
Davame, ale neprepisujeme stary kod. A vypli sme si enforcement tohto pravidla v nasom krasnom lintri, lebo to iritovalo head architekta ;-)

Reply:
Ja tu vidím diff kde sa zmenili

Reply:
Prauda, moja chyba. Tam je zjavne este dalsie pravidlo, ine ako pouzivanie jednoritych vs dvojitych. Vypinam aj toto

https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/QuotedSymbols